File: 01_Programming_Basics/01_Programming_Basics_with_C++/06_Nested_Conditional_Statements_Lab/03_Point_In_Rectangle.cpp
#include <iostream>
using namespace std;

int main() {
  double x1, y1, x2, y2, x, y;
  string msg;
  cin >> x1 >> y1 >> x2 >> y2 >> x >> y;

  if ((x >= x1 && x <= x2) && (y >= y1 && y <= y2)) {
    msg = "Inside";
  } else {
    msg = "Outside";
  }

  cout << msg << endl;

  return 0;
  
}

File: 01_Programming_Basics/03_Programming_Basics_with_JavaScript/04_Nested_Conditional_Statements/03_PointInRectangle.js
function PointInRectangle(args) {
  let x1 = parseFloat(args[0]);
  let y1 = parseFloat(args[1]);
  let x2 = parseFloat(args[2]);
  let y2 = parseFloat(args[3]);
  let x = parseFloat(args[4]);
  let y = parseFloat(args[5]);

  let insideX = false;
  let insideY = false;
  let msg = "";

  if (x >= x1 && x <= x2 ) {insideX = true;}
  if (y >= y1 && y <= y2 ) {insideY = true;}

  if (insideX == true && insideY == true) {
    msg = "Inside";
  } else {
    msg = "Outside";
  }
 
  console.log(msg);
}